src/charm.py

```diff
@@ -334,6 +395,16 @@ def render_manifest(self, manifest, name):
             yaml.safe_dump_all(manifest, f)
         return destination
 
+    def render_template(self, filename, **kwargs):
+
+        os.makedirs("templates/rendered", exist_ok=True)
+        destination = f"templates/rendered/{filename}_rendered.yaml"
```

Comment:
something feels awkward (and it could be me) for this to land in `/var/lib/juju/agents/unit-kube-ovn-*/charm/`

i guess it's not the worst thing have a `rendered/` folder, i've never thought about it before.

Reply:
We are using this folder to store the rendered templates/manifests. This is also used in the render_manifest method. Do you think it would be better to store them in a `tmp` folder?

Reply:
meh... i can bend on this pretty easily.  I'm not used to putting things in the charm folder when it's not part of the charm.  Folks come around later and find "rendered" folders and wonder where it comes from.  It makes it trickier to clean up later